KomaMRICore/test/Project.toml

```diff
@@ -1,6 +1,14 @@
 [deps]
 HDF5 = "f67ccb44-e63f-5c2f-98bd-6dc0ccc4ba2f"
+JLD2 = "033835bb-8acc-5ee8-8aae-3f567f8a3819"
+MAT = "23992714-dd62-5051-b70f-ba57cb901cac"
+MRIFiles = "5a6f062f-bf45-497d-b654-ad17aae2a530"
+Pkg = "44cfe95a-1eb2-52ea-b672-e2afdf69b78f"
+Reexport = "189a3867-3050-52da-a836-e630ba90ab69"
+Scanf = "6ef1bc8b-493b-44e1-8d40-549aa65c4b41"
 Suppressor = "fd094767-a336-5f1f-9728-57cf17d0bbfb"
 Test = "8dfed614-e22c-5e08-85e1-65c5234f0b40"
 TestItemRunner = "f8b46487-2199-4994-9208-9a1283c18c0a"
 TestItems = "1c621080-faea-4a02-84b6-bbd5e436b8fe"
+FileIO = "5789e2e9-d7fb-5bc7-8068-2c6fae9b9549"
```

Comment:
Remove additional packages

Reply:
Addressed in [a7145d6](https://github.com/cncastillo/KomaMRI.jl/commit/a7145d642a633d7aa5d238007b66511b91241eda)